### PR TITLE
RCv3 cafe convergence tests

### PR DESCRIPTION
**Pull request by cyli**
Some changes to make RCv3 cafe tests run better and in convergence.

~~Still waiting on the following:~~
- [x] #1675 
- [x] https://github.com/rackerlabs/autoscaling-chef/pull/763

#1681 Should also be done, but possibly as a different PR.

### CODE REVIEW

**Comment by cyli** [comment]
jenkins please run integration tests

File: autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py [hunk 1]
```diff
@@ -216,7 +196,7 @@ def test_create_nonzero_scaling_group_with_pool_on_cloud_network(self):
             asserter=self)
         self.verify_group_state(pool_group_resp.entity.id, self.min_servers)
 
-    @tags(speed='slow', type='rcv3', rcv3_mimic='fail')
+    @tags(speed='slow', type='rcv3', rcv3_mimic='fail', convergence='error')
```
Comment (manishtomar) [inline autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py]: Is it possible to describe or link to an issue that causes this test to fail?

**Comment by manishtomar** [comment]
LGTM modulo comment :)

File: autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py (hunk 1 above)
Comment (cyli) [inline autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py]: Oh good point, thanks.  :)  It's just that it's testing for desired capacity going down in response to an error.  I'll change these to 'never' instead, as well.

**Comment by cyli** [comment]
Thanks @manishtomar!  Merging.

**Pull request merged**